app/Http/Controllers/AuthenticationController.php

```diff
@@ -150,7 +159,7 @@ public function lookupEmail(LookupEmailRequest $request)
      * Show the account creation form
      * @param $token
      * @param Request $request
-     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View|\Illuminate\Http\RedirectResponse
```

Comment:
Nit: full namespace not needed here since it’s already imported. Would be nice to also get the other return types imported

Reply:
Thanks, I cleaned up the return types in the file.